handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -507,73 +497,74 @@ static long toBIO(PrivateKey key) throws Exception {
         if (key == null) {
             return 0;
         }
-        ByteBuf buffer = Unpooled.directBuffer();
+
+        ByteBufAllocator allocator = ByteBufAllocator.DEFAULT;
```

Comment:
I missed that it was used in 2 spots originally ... thanks for clarifying ... I'm fine leaving it as is.